src/hotspot/share/gc/shenandoah/shenandoahRootProcessor.cpp

```diff
@@ -189,36 +189,13 @@ ShenandoahRootScanner::~ShenandoahRootScanner() {
 }
 
 void ShenandoahRootScanner::roots_do(uint worker_id, OopClosure* oops) {
-  CLDToOopClosure clds_cl(oops, ClassLoaderData::_claim_strong);
+  CLDToOopClosure clds_cl(oops, ClassLoaderData::_claim_none);
```

Comment:
Are you sure that we don't have to claim CLDs? How would the threads avoid doubly-marking CLDs?

Reply:
Actually it looks like the CLD closure is not used at all, or is it?

Reply:
Right, we don't need to mark on-stack CLDs, concurrent mark CLDs should be sufficient.

Reran tests, still good.